src/components/Dashboard.test.js

```diff
@@ -1,8 +1,62 @@
+/** Test Suites and Cases for Dashboard component
+ *
+ *  @author Mayur Borse <mayur@hyphenos.io>
+ */
+
+import React from "react";
+import { shallow, configure } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+
+import Dashboard from "./Dashboard";
+import defaultConfig from "./constants/defaultConfig";
+
+configure({ adapter: new Adapter() });
+
 /**
- * Test cases for ensuring passed props are valid and doesn't crash the application
+ *  Test Suite with cases for ensuring passed props are valid and doesn't crash the application
  */
 describe("<Dashboard packets={packets} config={config}/>", () => {
-  it("Does not mount Table component if packets and config are undefined", () => {});
-  it("Mounts Table component if packets and config are defined", () => {});
-  it("Mounts Table component if packets are defined and config is undefined", () => {});
+  /**
+   *  Dashboard component should have a default config which is used if no config provided
+   */
+  it("Does not mount Table component if packets and config are undefined", () => {
+    const wrapper = shallow(<Dashboard />); // Equivalent to <Dashboard packets={undefined} config={undefined} />
+    expect(wrapper.exists("Table")).toEqual(false);
+  });
+
+  it("Mounts Table component if packets and config are defined", () => {
+    const wrapper = shallow(
+      <Dashboard packets={[null]} config={defaultConfig} />
+    );
+    expect(wrapper.exists("Table")).toEqual(true);
+  });
+
+  it("Mounts Table component (with default config) if packets are defined and config is undefined", () => {
```

Comment:
Check that is added to next test case about `defaultConfig` should be added here as well.